src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +126,64 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String salary} into an {@code Salary}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code salary} is invalid.
+     */
+    public static Salary parseSalary(String salary) throws ParseException {
+        requireNonNull(salary);
+        String trimmedSalary = salary.trim();
+        if (!Salary.isValidSalary(trimmedSalary)) {
+            throw new ParseException(Salary.MESSAGE_SALARY_CONSTRAINTS);
+        }
+        return new Salary(trimmedSalary);
+    }
+
+    /**
+     * Parses a {@code String employDate} into an {@code EmployDate}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code employDate} is invalid.
+     */
+    public static EmployDate parseEmployDate(String emplyDate) throws ParseException {
```

Comment:
Possible typo for `employDate`?

Reply:
`emplyDate` is indeed a typo. It has been corrected to `employDate` in the lastest commit.